index.html

```diff
@@ -5,11 +5,18 @@
 
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="description" content="">
-    <meta name="author" content="">
+    <meta name="description" content="Announcing the fourth Chaincode Labs residency, hosted in our office in Midtown Manhattan.">
+    <meta name="author" content="Chaincode Labs">
 
     <title>Chaincode Residency Summer 2019</title>
 
+    <meta property="twitter:title" content="Chaincode Labs Residency - Summer 2019">
+    <meta property="og:url" content="https://residency.chaincode.com">
+    <meta property="twitter:image:src" content="https://cdn-images-1.medium.com/max/1200/1*Gijq2XR7P-sfHf3xNvpRgg.png">
```

Comment:
I'm not sure how reliable this URL is. Anything with a random magic string seems a bit ephemeral and scary to me.

Perhaps add the medium image to the `img` directory of this repo and then set this content to `https://residency.chaincode.com/img/blah`?

Reply:
Fair. I did it for speed this morning, but you are right to call it out.

Reply:
Fixed here: https://github.com/chaincodelabs/summer-2019-residency-website/pull/8